### PR TITLE
[RHPAM-2830] 7.7.x BC and Kie Servers aren't syncronized 

### PR DESCRIPTION
**Pull request by desmax74**
Revert Kie Server strategy to restore HA behaviour on 7.7.x

https://issues.redhat.com/browse/RHPAM-2830

@spolti  @rhtevan @Sgitario @jakubschwan

### CODE REVIEW

**Comment by desmax74** [comment]
Waiting @rhtevan to apply 1 2 and 3 (if needed) in one commit

**Comment by rhtevan** [comment]
@desmax74 
#1. Doc updates need to ping @mramendi
#2. Correct, need to change it back to Rolling like this one (https://github.com/jboss-container-images/rhpam-7-openshift-image/blob/d401e3997b8f9f9c6c9689b83b306ad5d69430c1/templates/rhpam76-authoring-ha.yaml#L1122-L1126)
#3. Better to leave it as-is, it would be easier if user would like to switch again back to 'controllerbased' strategy.

File: templates/docs/rhpam77-authoring-ha.adoc [hunk 1]
```diff
@@ -66,6 +66,7 @@ https://docs.okd.io/latest/architecture/core_concepts/templates.html#parameters[
 |`KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED` | `KIE_SERVER_CONTROLLER_OPENSHIFT_GLOBAL_DISCOVERY_ENABLED` | If set to true, turns on KIE server global discovery feature (Sets the org.kie.server.controller.openshift.global.discovery.enabled system property) | false | False
 |`KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | `KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE` | Enables connection to KIE Server via OpenShift internal Service endpoint. (Sets the org.kie.server.controller.openshift.prefer.kieserver.service system property) | true | False
 |`KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | `KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL` | KIE ServerTemplate Cache TTL in milliseconds. (Sets the org.kie.server.controller.template.cache.ttl system property) | 5000 | False
+|`KIE_WORKBENCH_CONTROLLER_OPENSHIFT_ENABLED` | -- | true
```
Comment (Sgitario) [inline templates/docs/rhpam77-authoring-ha.adoc:69]: This line has broken the table. The table has 5 columns where the line only has 3 here. Also, the description should say "If set to true, turns on OpenShift integration feature (Sets the org.kie.server.controller.openshift.enabled system property)".

Comment (desmax74) [reply templates/docs/rhpam77-authoring-ha.adoc:69]: I'll remove and leave the adoc update procedure to update

Comment (rhtevan) [reply templates/docs/rhpam77-authoring-ha.adoc:69]: @Sgitario @spolti Will those adoc items being updated automatically by the build?

**Pull request merged**